arcade/arcade/cli/launcher.py

```diff
@@ -127,52 +134,47 @@ def _get_config_file(
         RuntimeError: If the config file is not found and is not optional.
     """
     if file_path:
-        config_path = Path(os.path.expanduser(file_path)).resolve()
-        if not config_path.is_file():
-            console.print(f"❌ Config file not found at {config_path}", style="bold red")
-            raise RuntimeError(f"Config file not found at {config_path}")
-        return str(config_path)
-
-    # Look for the file in the current working directory
-    config_path = Path(os.getcwd()) / default_filename
-    if config_path.is_file():
-        console.print(f"Using config file at {config_path}", style="bold green")
-        return str(config_path)
-
-    # Look for the file in the user's home directory under .arcade/
-    home_config_path = Path.home() / ".arcade" / default_filename
-    if home_config_path.is_file():
-        console.print(f"Using config file at {home_config_path}", style="bold green")
-        return str(home_config_path)
-
-    # Look for known installation directories
-    for path in known_engine_config_locations:
-        etc_path = Path(path) / default_filename
-        if etc_path.is_file():
-            console.print(f"Using config file at {etc_path}", style="bold green")
-            return str(etc_path)
+        candidate = Path(os.path.expanduser(file_path)).resolve()
+        if not candidate.is_file():
+            console.print(f"❌ Config file not found at {candidate}", style="bold red")
+            raise RuntimeError(f"Config file not found at {candidate}")
+        return str(candidate)
+
+    # List of all config file path locations to check.
+    candidates = [
+        Path(os.getcwd()) / default_filename,
+        Path.home() / ".arcade" / default_filename,
+    ]
+    candidates.extend(Path(path) / default_filename for path in known_engine_config_locations)
```

Comment:
Do we have a similar pattern for the env file? Its installed in the same dir as the engine config

Reply:
yes, but since it is technically optional it will display as a warning if it can't be found.

Output:
![image](https://github.com/user-attachments/assets/48a96b1b-b106-4ccc-9629-cb4b42596149)